### DISCUSSION TITLE
Candidate packages

### DISCUSSION THREAD

**Discussion by jamesdbrock** [haskell-github-trust/.github#3] [General]
Notes on some trustees and packages to invite.

**Comment by jamesdbrock** [comment]
Packages from @sjakobi https://github.com/haskellfoundation/stability/pull/12#issuecomment-1120401414

**Comment by jamesdbrock** [comment]
Package from @liyang and @fumieval https://github.com/fumieval/thyme

**Comment by ad-si** [comment]
https://github.com/ad-si/ulid

**Reply by jamesdbrock** [reply]
Sent you an ownership invite.

**Comment by jamesdbrock** [comment]
https://github.com/timbod7/haskell-chart/issues/243

**Comment by jamesdbrock** [comment]
This one is already in an org https://github.com/gtk2hs/gtk2hs/issues/323

**Comment by ad-si** [comment]
[meteogrid/iso8601-duration](https://github.com/meteogrid/iso8601-duration) by @meteogrid

**Comment by mwotton** [comment]
this is a really good project that only required a few updates to get it working again, but the maintainer hasn't replied in 3 weeks - is there a general consensus on how long we ought to wait? https://github.com/goodlyrottenapple/quickjs-hs/pull/2

**Reply by jamesdbrock** [reply]
No commits to that project in 3 years... I'd say that's an excellent candidate. I invited you to be an Owner. You can invite @goodlyrottenapple to be an Owner too if you want.

If you can't get any reply from @goodlyrottenapple then you could try to take over the package to get it into the Trust. https://github.com/haskell-github-trust#how-to-take-over-a-package

**Reply by goodlyrottenapple** [reply]
Hey, sorry for not replying. Hadn't worked on this since initial release and didn't notice @mwotton's PR. Happy for anyone who finds this library useful to take over.

**Reply by mwotton** [reply]
@goodlyrottenapple could you add me as a maintainer on hackage, then, and i'll transfer it to the trust? or you can transfer it directly if you prefer, don't mind.

**Reply by jamesdbrock** [reply]
Here's how to add your package to the Haskell Github Trust @goodlyrottenapple : https://github.com/haskell-github-trust#how-to-add-your-own-package-to-the-trust

**Reply by goodlyrottenapple** [reply]
Happy to do it. Could I be added as a trust owner so that I can transfer the repo?

**Reply by jamesdbrock** [reply]
@goodlyrottenapple Sorry for the delayed reply, I just sent you an ownership invitation.

**Comment by jamesdbrock** [comment]
https://discourse.haskell.org/t/deciding-whether-to-create-a-new-package-or-take-over-an-abandoned-one/6285

https://hackage.haskell.org/package/clippings

**Comment by mwotton** [comment]
Network-uri-json might be a good one

**Comment by jamesdbrock** [comment]
https://www.reddit.com/r/haskell/comments/13g804o/comment/jkqt7t9/?utm_source=share&utm_medium=web2x&context=3

https://hackage.haskell.org/package/wx

https://hackage.haskell.org/package/Euterpea

**Comment by fumieval** [comment]
https://hackage.haskell.org/package/fernet
